dask/array/tests/test_reductions.py

```diff
@@ -67,6 +67,8 @@ def test_reductions_1D(dtype):
     reduction_1d_test(da.nansum, a, np.nansum, x)
     with ignoring(AttributeError):
         reduction_1d_test(da.nanprod, a, np.nanprod, x)
+        reduction_1d_test(da.nancumprod, a, np.nancumprod, x)
```

Comment:
These unit tests aren't actually being run unless dask is testing against numpy master. That *might* be a good idea, actually, but in the meantime you might want to add a backport to dask, too.